frappe_graphql/frappe_graphql/mutations/delete_doc.py

```diff
@@ -0,0 +1,19 @@
+from graphql import GraphQLSchema, GraphQLResolveInfo
+
+import frappe
+
+
+def bind(schema: GraphQLSchema):
+    schema.mutation_type.fields["deleteDoc"].resolve = delete_doc_resolver
+
+
+def delete_doc_resolver(obj, info: GraphQLResolveInfo, **kwargs):
+    doctype = kwargs["doctype"]
```

Comment:
For the sake of consistency, I reckon it's better to use `.get()` for accessing dict values

Reply:
@assemmarwan updated 👍